spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /index' do
+    it 'returns http success' do
+      get '/posts/index'
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe 'GET /show' do
+    it 'returns http success' do
+      get '/posts/show'
+      expect(response).to have_http_status(:success)
+    end
```

Comment:
- [ ]Kindly make sure to implement the required tests for each of the actions (three tests per action), as per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/controller_specs_project.md#:~:text=If%20response%20status,correct%20placeholder%20text.) requirement.